### ISSUE TITLE
  e ) erase_clusters=false ;;... doesnt work

### ISSUE THREAD

**Issue by jayunit100** [kubernetes-sigs/kpng#352]
hey yall im trying to 

- hack up the test_e2e.sh script to do `erase_clusters=false` but im not seeing any effect...

```

while getopts "i:b:B:cdD:eE:n:ps:" flag
do
    case "${flag}" in
        i ) ip_family="${OPTARG}" ;;
        b ) backend="${OPTARG}" ;;
        c ) ci_mode=true ;;
        d ) devel_mode=true ;;
        e ) erase_clusters=false ;;
```

but then if i run 

```
eval "$(curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | GIMME_GO_VERSION=1.18beta1 bash)" go version go1.18beta1 linux/amd64
./test_e2e.sh -i ipv4 -b iptables 


```

I ultimately end up with cluster gettin deleteddd :( :( :( 

```
+==================================================================+
                Erasing kind clusters
+==================================================================+
[ PASSED ] Cluster kpng-e2e-ipv4-iptables deleted.
```

/good-first-issue

**Comment by emabota** [comment]
Hi @jayunit100, 
I am a New Contributor and would like to work on this issue. Can you please provide some orientation on how to approach this issue? Thanks 
/assign

**Comment by jayunit100** [comment]
Hi just try to run the hack/test_e2e script w iptables option and use the -e flag you will see what I mean :)

**Comment by emabota** [comment]
Hi @jayunit100, 
while trying to replicate this issue, whenever I run the script `./test_e2e.sh -i ipv4 -b iptables` it fails with the following end message: 

> ERROR: unknown cluster "[ \x1b[34mINFO\x1b[0m ] kpng-e2e-ipv4-iptables" 

Can you, please, help me figure out why it's happening? Find attached the "log" of the script.
[test_e2e_status.log](https://github.com/kubernetes-sigs/kpng/files/10114534/test_e2e_status.log)

**Comment by jayunit100** [comment]
Ouch looks like you didn't get a cluster is kind installed ? Any other suspicious logs ?

**Comment by emabota** [comment]
This was fixed here #399 
> Hi @jayunit100, while trying to replicate this issue, whenever I run the script `./test_e2e.sh -i ipv4 -b iptables` it fails with the following end message:
> 
> > ERROR: unknown cluster "[ \x1b[34mINFO\x1b[0m ] kpng-e2e-ipv4-iptables"
> 
> Can you, please, help me figure out why it's happening? Find attached the "log" of the script. [test_e2e_status.log](https://github.com/kubernetes-sigs/kpng/files/10114534/test_e2e_status.log)
